src/common/utils.js

```diff
@@ -60,17 +60,23 @@ function isValidHexColor(hexColor) {
 }
 
 /**
- * @param {string} value
- * @returns {boolean | string}
+ * @param {string | boolean} value
+ * @returns {boolean | undefined }
  */
 function parseBoolean(value) {
-  if (value === "true") {
-    return true;
-  } else if (value === "false") {
-    return false;
-  } else {
+  if (typeof value === "boolean") {
```

Comment:
What's this line meant to do?

Reply:
@anuraghazra, thanks for reviewing this PR! I added that condition since the tests were failing. I, however, understand that it is a bit confusing given the type definition says that the function only takes strings. Alternatively, I can change all the `true` parameters in the tests to true strings.

Reply:
The function now implements he following behaviour:
- `true` -> `true`
- `false` -> `false`
- true, True, TRUE -> `true`
- false, False, FALSE -> `false`
- Random string -> `undefined`